src/ucc/api/ucc.h

```diff
@@ -1712,12 +1712,21 @@ typedef enum {
                                                             Note, the status is not guaranteed
                                                             to be global on all the processes
                                                             participating in the collective.*/
-    UCC_COLL_ARGS_FLAG_MEM_MAPPED_BUFFERS   = UCC_BIT(7)  /*!< If set, both src
+    UCC_COLL_ARGS_FLAG_MEM_MAPPED_BUFFERS   = UCC_BIT(7), /*!< If set, both src
                                                             and dst buffers
                                                             reside in a memory
                                                             mapped region.
                                                             Useful for one-sided
                                                             collectives. */
+    UCC_COLL_ARGS_FLAG_ONESIDED_VECTOR      = UCC_BIT(8)  /*!< If set, displacements and 
+                                                               counts for vector collectives
+                                                               (e.g., AlltoallV), will be interpreted
+                                                               differently. Entry i in the src displacement,
+                                                               will be the displacement relative to the 
+                                                               start address for outgoing data to team member
+                                                               i. Entry i in the dst displacement will be 
+                                                               the displacement relative to the dst buffer 
+                                                               of team member i from team member j */
```

Comment:
The flag was meant to change the semantics of the displacements in the coll_args, but not sure this flag is necessary enough to do this. We decided to remove the flag as there are some implicit flags already indicating we will be performing a onesided alltoallv (i.e., MEM_MAP_BUFFERS flag and GLOBAL_WORK_BUFFER mask). In addition, the user must explicit indicate they want to use this algorithm rather than a two-sided algorithm.